experimental/wgit/weigit_api.py

```diff
@@ -14,92 +16,179 @@
 
 
 class WeiGit:
```

Comment:
I am thinking that we should combine the WeiGit class and the Repo class. Just use the repo class for everything that WeiGit class is doing. I.e. 

repo = Repo()
repo.add()
repo.status()

etc.

What's the reason to have 2 separate classes?